katsdpimageutils/primarybeamcorrection/primary_beam_correction.py

```diff
@@ -152,13 +159,15 @@ def standard_deviation(data):
 
 
 def inverse_variance(data):
-    """ Calculate the inverse variance. Reject pixels more than 5 sigma from
-    the median and reject all zeros until either no more pixels are rejected or
-    a maximum  of 50 iterations is reached.
+    """Calculate the inverse variance.
+
+    Reject pixels more than 5 sigma from the median and reject all zeros
+    until either no more pixels are rejected or a maximum of 50 iterations
+    is reached.
     """
     data = data[data != 0.0]
     if len(data) == 0:
-        return float(0.0), float(0.0)
+        return float(0.0)
```

Comment:
```suggestion
        return 0.0
```
0.0 is already a float.